src/lib/frontend/d_cnf.ml

```diff
@@ -200,11 +202,11 @@ let builtin_enum = function
   | Ty.Tsum (name, cstrs) as ty_ ->
     let ty_cst =
       DStd.Expr.Id.mk ~builtin:B.Base
-        (DStd.Path.global (Hstring.view name))
+        (DStd.Path.global (Uid.show name))
         DStd.Expr.{ arity = 0; alias = No_alias }
```

Comment:
This looks a bit weird; aren't we building a second Dolmen identifier with the same name as an existing one here?

Reply:
Indeed, I thought it was correct but we duplicate the identifier here. I fixed the issue by creating the constructors and 
the Dolmen type in `Fpa_rounding`.